File: mpp/LabMPPCurse/Networking/src/main/java/dto/PersonalDTO.java
package dto;

import java.io.Serializable;

public class PersonalDTO  implements Serializable {

    private String userName;
    private String pass;

    public PersonalDTO(String username, String password) {
        this.userName=username;
        this.pass=password;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getPass() {
        return pass;
    }

    public void setPass(String pass) {
        this.pass = pass;
    }
}

File: mpp/LabMppCurseWPF/Networking/DTOs/PersonalDTO.cs
using System;

namespace Networking.DTOs
{	
	[Serializable]
	public class PersonalDTO
	{
		private string _userName;
		private string _passwd;

		public PersonalDTO(string userName, string passwd)
		{
			_userName = userName;
			_passwd = passwd;
		}

		public virtual string UserName
		{
			get { return _userName; }
			set { _userName = value; }
		}

		public virtual string Passwd
		{
			get { return _passwd; }
			set { _passwd = value;  }
		}

	}
}